src/cryptoadvance/specter/util/bitcoind_setup_tasks.py

```diff
@@ -0,0 +1,212 @@
+import os, time, requests, secrets, platform, tarfile, zipfile, sys
+from ..bitcoind import BitcoindPlainController
+import pgpy
+from pathlib import Path
+from .sha256sum import sha256sum
+
+
+def setup_bitcoind_thread(specter=None):
+    try:
+        BITCOIND_OS_SUFFIX = {
+            "Windows": "win64.zip",
+            "Linux": "x86_64-linux-gnu.tar.gz",
+            "Darwin": "osx64.tar.gz",
+        }
+        bitcoind_url = f"https://bitcoincore.org/bin/bitcoin-core-0.21.0/bitcoin-0.21.0-{BITCOIND_OS_SUFFIX[platform.system()]}"
```

Comment:
Maybe we can pass in the version as a param to setup_bitcoind_thread ? At best i'd like to see the version configured in config.py and there something like (BaseConfig?!)
```
INTERNAL_BITCOIND_VERSION = os.getenv("INTERNAL_BITCOIND_VERSION", "bitcoin-core-0.21.0")
```
and then calling it like 
```
args=(app.specter,app.config["INTERNAL_BITCOIND_VERSION"],),
```

Reply:
I think it would make more sense to make it part of the specter config than the app config no?

Reply:
On second thought, I guess it'd make sense to have this in the app, but save in specter config the currently installed version

Reply:
Maybe we want that more dynamic in the future so it's probably a good idea to store it in the specter-config (especially as i've recommended making it overridable per envvar) but for now i think it's better to have upgrades as part of our release-process.